### PR TITLE
Add livy submit

### PR DESCRIPTION
**Pull request by ericdill**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@AlbertDeFusco can you comment here that you're ok being a recipe maintainer?

### CODE REVIEW

**Comment by ericdill** [comment]
@conda-forge/help-python this is ready for review, thanks!

**Comment by npavlovikj** [comment]
The recipe LGTM, I just have few clean-up comments. Let's hear from @AlbertDeFusco if he is willing to be listed as a maintainer, and we can merge afterwards.

File: recipes/livy-submit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  sha256: 6e0d0a587d7787afce855d7f91bab77f293885b4dede7aae1b34348fde6bb77a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - livy = livy_submit.cli:cli
+
+
```
Comment (npavlovikj) [inline recipes/livy-submit/meta.yaml]: Remove extra empty line.

File: recipes/livy-submit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pexpect  
+    - python-hdfs
+    - requests
+    - requests-kerberos
+test:
```
Comment (npavlovikj) [inline recipes/livy-submit/meta.yaml]: Add empty line before `test`.

File: recipes/livy-submit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - livy submit -h
+    - livy info -h
+    - livy kill -h
+    - livy log -h
+
+about:
+  home: https://github.com/Anaconda-Platform/livy-submit
+  license: BSD 3-Clause
+  license_file: LICENSE
+  summary: CLI for submitting batch spark jobs to livy. Think spark-submit
```
Comment (npavlovikj) [inline recipes/livy-submit/meta.yaml]: Add `license_family: BSD`.

**Comment by AlbertDeFusco** [comment]
I am willing to be a maintainer.

File: recipes/livy-submit/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  commands:
+    - livy submit -h
+    - livy info -h
+    - livy kill -h
+    - livy log -h
+
+about:
+  home: https://github.com/Anaconda-Platform/livy-submit
+  license: BSD 3-Clause
+  license_file: LICENSE
```
Comment (npavlovikj) [inline recipes/livy-submit/meta.yaml]: Add `license_family: BSD`.

**Comment by ericdill** [comment]
Thanks @npavlovikj !

**Comment by npavlovikj** [comment]
Thank you for contributing @ericdill 🎉 `noarch` recipes are build only on Linux, so we can ignore the AppVeyor failure.

**Pull request merged**